taxonomy/algolia/constants.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Constants for algolia.
+"""
+# default configuration for the index
+ALGOLIA_INDEX_SETTINGS = {
+    'attributeForDistinct': 'external_id',
+    'distinct': True,
+    'typoTolerance': False,
+    'searchableAttributes': [
+        'unordered(name)',
+        'skills.name',
+    ],
+    'attributesForFaceting': [
+        'searchable(name)',
+        'searchable(skills.name)',
+    ],
+}
```

Comment:
any plan for adding the jobs data settings?

Reply:
These settings are for jobs index, I will rename to constant to be more explicit.